sovrin_client/agent/walleted.py

```diff
@@ -97,6 +103,17 @@ def __init__(self,
         }
         self.logger = logger
 
+        self.issuer_backend = None
+
+        self._invites = {}  # type: Dict[Nonce, Tuple(InternalId, str)]
+        self._attribDefs = {}  # type: Dict[str, AttribDef]
+        self.defined_claims = []  # type: List[Dict[str, Any]
+
+        self.available_claims_by_internal_id = {}  # type: Dict[InternalId, Set[ID]]
```

Comment:
Not all of these are issuer related. In fact _proofRequestsSchema don't seem to have anything to do with an issuer?